src/config.ts

```diff
@@ -387,6 +405,26 @@ export const configure = async (customProjectConfig?: CustomProjectConfig) => {
 
   const projectConfig = await loadProjectConfig();
 
+  if (
+    projectConfig.storybookUrl &&
+    (projectConfig.pages || projectConfig.pageBaselineUrl)
+  ) {
+    log(
+      "Currently storybook & page screenshot modes can't be used together. Either set storybookUrl or pages & pageBaselineUrl",
+    );
+    process.exit(1);
+  }
+
+  if (
+    (projectConfig.pages && !projectConfig.pageBaselineUrl) ||
+    (!projectConfig.pages && projectConfig.pageBaselineUrl)
+  ) {
+    log(
+      'Page screenshot mode requires both pages & pageBaselineUrl config properties.',
+    );
+    process.exit(1);
+  }
+
```

Comment:
for our javascript friends - so that they have a clue why lost-pixel is not working :D